nuprojs/Microsoft.ServiceFabric.AspNetCore.Abstractions/Microsoft.ServiceFabric.AspNetCore.Abstractions.nuproj

```diff
@@ -14,20 +14,133 @@
     <Import Project="$(NuProjPath)\NuProj.targets" />
 
     <ItemGroup>
-      <File Include="$(DropFolderNetStandard)Microsoft.ServiceFabric.AspNetCore.dll">
-        <TargetPath>lib\netstandard2.0</TargetPath>
+      <File Include="$(DropFolderNetFramework)\Microsoft.ServiceFabric.AspNetCore.dll">
+        <TargetPath>lib\net461</TargetPath>
       </File>
-      <File Include="$(DropFolderNetStandard)Microsoft.ServiceFabric.AspNetCore.xml">
-        <TargetPath>lib\netstandard2.0</TargetPath>
+      <File Include="$(DropFolderNetFramework)\Microsoft.ServiceFabric.AspNetCore.xml">
```

Comment:
Net461 should be sufficient I believe. This nuproj is modelled this way to remain consistent with other nuprojs in winfab/actors repo
For versioning and backward compatibility - ref https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/version-compatibility#version-compatibility-for-components